src/olympia/lib/settings_base.py

```diff
@@ -604,133 +604,10 @@ def get_db_config(environ_var, atomic_requests=True):
 # and js files that can be bundled together by the minify app.
 MINIFY_BUNDLES = {
```

Comment:
This is unused now?  If so, can we just delete the setting entirely?

Reply:
There's a final clean up PR that will come next.